### PR TITLE
docs(dynamic-forms): add the multiple feature from select in demo

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Move the `multiple` mat-select feature from the experimental app into the documentation and demo.

### What's included?

- updated README
- updated Demo

#### Test Steps

- [ ] `npm run serve`
- [ ] Go to http://localhost:4200/#/components/dynamic-forms
- [ ] See select demos with multiple addition

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

File: src/app/components/components/dynamic-forms/dynamic-forms.component.ts [hunk 1]
```diff
@@ -124,6 +124,12 @@ export class DynamicFormsDemoComponent {
     type: TdDynamicElement.Select,
     selections: ['Test1', 'Test2', 'Test3', 'Test4'],
     required: true,
+  }, {
+    name: 'required-miltiple-select',
```
Comment (jeremysmartt) [inline src/app/components/components/dynamic-forms/dynamic-forms.component.ts]: misspelled multiple

Comment (JulieMarie) [reply src/app/components/components/dynamic-forms/dynamic-forms.component.ts]: *miltiple*

Comment (emoralesb05) [reply src/app/components/components/dynamic-forms/dynamic-forms.component.ts]: thats how i wanted to spell it

Comment (jeremysmartt) [reply src/app/components/components/dynamic-forms/dynamic-forms.component.ts]: 🤔

**Pull request merged**